lib/mindwendel_web/components/core_components.ex

```diff
@@ -380,8 +384,8 @@ defmodule MindwendelWeb.CoreComponents do
         name={@name}
         class={[
           "form-control",
-          @errors == [] && "is-valid",
-          @errors != [] && "is-invalid"
+          @used? && @errors == [] && "is-valid",
+          @used? && @errors != [] && "is-invalid"
```

Comment:
technically the 2nd check here isn't needed (as the function above already sets the errors to empty if it's unused) but I thought semantically it made sense